packages/schemas/src/FormsSettings/example.json

```diff
@@ -48,6 +48,13 @@
                 "Sobota 18-22": 24,
                 "Niedziela 10-14": 25
               }
+            },
+            {
+              "label": "Maksymalna graczy",
```

Comment:
Shouldn't label be "Maksymalna liczba graczy" here?

Reply:
😅